packages/easy-coding-standard/build/target-repository/.github/workflows/test.yaml

```diff
@@ -0,0 +1,23 @@
+name: Fixer Test
+
+on:
+    pull_request: null
+    push:
+        branches:
+            - main
+
+jobs:
+    fixer_test:
+        runs-on: ubuntu-latest
+
+        steps:
+            -   uses: actions/checkout@v2
```

Comment:
This would run `git clone ...`, which is not what you want to test. So should be removed

Reply:
hmm... is it okay to simply remove this or should it be replaces by something else?